From c6662f914013d3423130f0ad8c7b2e443613af97 Mon Sep 17 00:00:00 2001
From: Zachary Powell <zacthespack@gmail.com>
Date: Wed, 1 May 2019 16:40:03 +0100
Subject: [PATCH] Fixed reportlab import

**Describe the bug**
When running html generation I ended up with the error:

```
Exception occurred:
File "/usr/local/lib/python2.7/dist-packages/rst2pdf/flowables.py", line 895, in <module>
   if reportlab.Version == '2.1':
 NameError: name 'reportlab' is not defined
```

**To Reproduce**
Steps to reproduce the behavior:
```
sphinx-build -b html -d stage/doctrees   source/welcome stage/html
```
---
 rst2pdf/flowables.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/rst2pdf/flowables.py b/rst2pdf/flowables.py
index e357ac10..39342695 100644
--- a/rst2pdf/flowables.py
+++ b/rst2pdf/flowables.py
@@ -5,6 +5,7 @@
 
 from copy import copy
 import re
+import reportlab
 
 from reportlab.platypus import *
 from reportlab.platypus.doctemplate import *
